packages/ketcher-core/src/application/render/restruct/reatom.ts

```diff
@@ -648,7 +648,7 @@ function shouldDisplayStereoLabel(
 }
 
 function isLabelVisible(restruct, options, atom) {
-  const isAttachmentPointAtom = atom.a.attpnt != null;
+  const isAttachmentPointAtom = atom.a.attpnt;
```

Comment:
Could I propose to use this approach: 

```javascript
= Boolean(atom.a.attpnt)
``` 

(in this case the variable `isAttachmentPointAtom` will store the boolean type but not one of three: null, undefined, 0).